packages/flutter/lib/src/widgets/transitions.dart

```diff
@@ -284,10 +284,15 @@ class ScaleTransition extends AnimatedWidget {
 
   @override
   Widget build(BuildContext context) {
+    /// The ImageFilter layer created by setting filterQuality will introduce
+    /// a saveLayer call. This is usually worthwhile when animating the layer,
+    /// but leaving it in the layer tree before the animation has started or after
+    /// it has finished significantly hurts performance.
```

Comment:
Should we leave a note about this behavior in the doc comment on `filterQuality`?

Reply:
Done